Octokit.Tests/Models/IssueTest.cs

```diff
@@ -257,7 +257,7 @@ public void CreatesAnIssueUpdateRequestObject()
 
             var update = issue.ToUpdate();
 
-            Assert.Null(update.Labels);
+            Assert.NotNull(update.Labels);
```

Comment:
Apologies for the drive-thru comment, but maybe what we should actually have is a standard group of `SetLabel`/`AppendLabel`/`ClearLabels` methods to clearly allow for all the possible ways of manipulating the list (replacing/appending/clearing) so that users won't be tempted to manipulate the list directly and can trust the methods to do what is required? And then apply the same pattern to all multiple categorization lists (assignees, etc).